### PR TITLE
10-10CG Caregiver A11y Findings

### PR DESCRIPTION
**Pull request by LinnJS**
## Description
Fix issues documented in [Issue #9342](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9342)

## Acceptance criteria
- [x] Issues from [Issue #9342](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9342) are resolved

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/caregivers/definitions/content.js [hunk 1]
```diff
@@ -22,18 +22,21 @@ export const links = {
     label: 'Get help filing your claim.',
     phoneLink: 'tel:18552603274',
     phoneLabel: '855-260-3274',
+    phoneAriaLabel: '8 5 5. 2 6 0. 3 2 7 4.',
```
Comment (LinnJS) [inline src/applications/caregivers/definitions/content.js:25]: Content for `aria-label` attribute

File: src/applications/caregivers/containers/ConfirmationPage.jsx [hunk 2]
```diff
@@ -18,7 +18,11 @@ const ConfirmationPage = props => {
 
   const PrintDetails = () => (
     <div className="print-details">
-      <img src="/img/design/logo/va-logo.png" alt="VA logo" width="300" />
+      <img
+        src="/img/design/logo/logo-black-and-white.png"
```
Comment (LinnJS) [inline src/applications/caregivers/containers/ConfirmationPage.jsx:22]: black and white VA logo

File: src/applications/caregivers/config/form.js [hunk 3]
```diff
@@ -107,7 +107,7 @@ const formConfig = {
       pages: {
         veteranInfoOne: {
           path: 'vet-1',
-          title: ' ',
+          title: 'Veteran information',
```
Comment (LinnJS) [inline src/applications/caregivers/config/form.js:110]: Add titles for the edit labels on the review page

File: src/applications/caregivers/components/PreSubmitInfo/components/SignatureInput/index.jsx [hunk 4]
```diff
@@ -31,20 +31,18 @@ const SignatureInput = ({ fullName, required, label, setIsSigned }) => {
...
-        required={required}
-        onValueChange={value => setSignature(value)}
-        field={{ value: signature.value, dirty: signature.dirty }}
-        errorMessage={
-          hasError &&
-          'Your signature must match your first and last name as previously entered.'
-        }
-      />
-    </>
+    <ErrorableTextInput
```
Comment (LinnJS) [inline src/applications/caregivers/components/PreSubmitInfo/components/SignatureInput/index.jsx:34]: Remove the external label and use the label internal to the formation input component

File: src/applications/caregivers/components/AdditionalInfo/index.jsx [hunk 5]
```diff
@@ -5,7 +5,7 @@ import { links } from 'applications/caregivers/definitions/content';
 
 const VetInfo = ({ pageTitle, headerInfo }) => (
   <>
-    {pageTitle && <h5 className="vads-u-font-size--h4">{pageTitle}</h5>}
+    {pageTitle && <h3 className="vads-u-font-size--h4">{pageTitle}</h3>}
```
Comment (LinnJS) [inline src/applications/caregivers/components/AdditionalInfo/index.jsx:8]: Fix heading hierarchy

**Review by bradl3yC — APPROVED** [review]
LGTM nice work

**Review by Mottie — APPROVED** [review]

File: yarn.lock [hunk 6]
```diff
@@ -1014,7 +1013,7 @@
   dependencies:
     regenerator-runtime "^0.13.2"
 
-"@babel/template@^7.1.0", "@babel/template@^7.1.2", "@babel/template@^7.2.2":
+"@babel/template@^7.1.0":
```
Comment (U-DON) [inline yarn.lock:1016]: What updated the lock file?

**Review by U-DON — APPROVED** [review]

File: yarn.lock (hunk 6 above)
Comment (LinnJS) [reply yarn.lock:1016]: I think I just ran yarn install after merging master, should I revert?

**Pull request merged**